stable/polaris/templates/mutate-webhook.configuration.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if and .Values.webhook.enable .Values.webhook.enableMutation -}}
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: polaris-mutate-webhook
+  {{- if not .Values.webhook.secretName }}
+  annotations:
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ include "polaris.fullname" . }}-cert
+  {{- end }}
+webhooks:
+- admissionReviewVersions:
+  - v1
+  - v1beta1
+  clientConfig:
+    {{- if .Values.webhook.caBundle }}
+    caBundle: {{ .Values.webhook.caBundle | quote }}
+    {{- end }}
+    service:
+      name: {{ include "polaris.fullname" . }}-webhook
+      namespace: {{ .Release.Namespace }}
+      path: /mutate
+      port: 443
+  name: polaris.fairwinds.com
```

Comment:
The [mutating Webhooks spec](https://kubernetes.io/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1/) looks like it also supports the `failurePolicy` and `matchPolicy` present in our ValidatingWebhookConfiguration. Should we also support these for the mutating Webhook (using either re-using the validating Webhook values, or adding new chart values for the mutating Webhook)?